infrastructure/cdn-in-a-box/traffic_stats/Dockerfile-influxdb

```diff
@@ -18,7 +18,7 @@
 FROM influxdb:1.7.3
 
 RUN apt-get update && \
-    apt-get install -y dnsutils net-tools gettext-base netcat && \
+    apt-get --no-install-recommends install -y dnsutils net-tools gettext-base netcat && \
```

Comment:
See above—I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.